sdks/java/maven-archetypes/examples/sample.txt

```diff
@@ -0,0 +1,117 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
```

Comment:
Is it correct to put a license header on this? I think this might qualify for excluding from the RAT checker (I'm assuming we have an allow list somewhere)

Reply:
Yes, there is an allowlist. Here: https://github.com/apache/beam/blob/master/build.gradle#L37

Reply:
I originally left the license off on purpose. It seems like other Apache projects mostly leave the license off of example data, e.g. [apache/spark/blob/master/examples/src/main/resources/kv1.txt](https://github.com/apache/spark/blob/master/examples/src/main/resources/kv1.txt). How would I add the sample to the allow list?

Reply:
If we are switching to use --inputFile=/path/to/inputfile. I guess we can remove the sample completely.

If you woud like to add an exception you can add a line (https://github.com/apache/beam/blob/master/build.gradle#L42) pointing to the sample file.